back/engines/commercial/user_custom_fields/app/models/user_custom_fields/representativeness/ref_distribution.rb

```diff
@@ -44,6 +45,17 @@ def validate_distribution_options
     errors.add(:distribution, 'options must be a subset of the options of the associated custom field.')
   end
 
+  def validate_distribution_counts
+    return if custom_field.blank? || distribution.blank?
+
+    counts = distribution.values
+    errors.add(:distribution, 'population counts cannot be nil.') if counts.any?(&:nil?)
+
+    counts.compact!
+    errors.add(:distribution, 'population counts must be strictly positive.') unless counts.all?(&:positive?)
```

Comment:
Ok, good to know that values cannot be zero- forgot about this case, will also update this on the FE

Reply:
Yeah I thought having both 0 and the toggle would be confusing for everyone, including me 😅